bin/drone/shell

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+USAGE="
+SSH-es you into a live drone build, for interactive debugging.
+
+Usage: bin/drone/shell ec2-34-201-165-204.us-west-2.compute.amazonaws.com
+
+Notes:
+  1. Hostname is printed by the 'bin/drone/shell' step of the drone build.
+  2. Run only when drone is in either the unit->\`unit-tests\` or ui->\`ui-tests\` step,
```

Comment:
When is the latest this can be run?

Reply:
As long as its still in the `unit-tests` or `ui-tests` (main testing) step it can be run. As soon as that step ends, the container is killed, UNLESS you are already shelled in, in which case it stays running for another ... hour or two? Or until you log out.